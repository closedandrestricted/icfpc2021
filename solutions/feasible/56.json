{"vertices":[[26,51],[8,29],[41,75],[2,28],[29,51],[23,53],[41,77],[24,23],[35,23],[2,31],[31,49]]}