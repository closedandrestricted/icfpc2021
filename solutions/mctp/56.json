{"vertices":[[9,16],[37,19],[6,44],[44,78],[25,63],[34,47],[53,58],[29,91],[12,70],[46,75],[24,65]]}