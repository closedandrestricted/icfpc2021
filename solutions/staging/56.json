{"vertices":[[40,32],[51,6],[20,52],[15,53],[42,67],[40,32],[17,80],[15,76],[10,20],[18,53],[24,51]]}