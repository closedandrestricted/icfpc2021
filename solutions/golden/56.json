{"vertices":[[37,36],[59,18],[19,58],[46,5],[14,84],[37,36],[26,58],[29,60],[14,14],[9,48],[54,69]]}